resources/assets/js/validations.js

```diff
@@ -159,246 +144,231 @@ $.extend(true, laravelValidation, {
+        Integer: function(value) {
             var regex= new RegExp("^(?:-?\\d+)$",'i');
-            return this.optional(element) ||  regex.test(value);
-        }, $.validator.format("The field must be must be a integer."));
+            return  regex.test(value);
+        },
 
         /**
          * Validate that an attribute is numeric.
          */
-        $.validator.addMethod("laravelNumeric", function(value, element, params) {
-            return this.optional(element) ||
-                $.validator.methods.number.call(this, value, element, true);
-        }, $.validator.format("The field must be must be a number."));
+        Numeric: function(value, element) {
+            return $.validator.methods.number.call(this, value, element, true);
+        },
 
         /**
          * Validate that an attribute is a string.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelString", function(value, element, params) {
-            return this.optional(element) ||
-                typeof value == 'string';
-        }, $.validator.format("The field must be string"));
+        String: function(value) {
+            return typeof value === 'string';
+        },
 
         /**
          * The field under validation must be numeric and must have an exact length of value.
          */
-        $.validator.addMethod("laravelDigits", function(value, element, params) {
-            return this.optional(element) ||
-                ($.validator.methods.number.call(this, value, element, true)
+        Digits: function(value, element, params) {
+            return ($.validator.methods.number.call(this, value, element, true)
                 && value.length==params);
-        }, $.validator.format("The field must be {0} digits."));
+        },
 
         /**
          * The field under validation must have a length between the given min and max.
          */
-        $.validator.addMethod("laravelDigitsBetween", function(value, element, params) {
-            return this.optional(element) ||
-                ($.validator.methods.number.call(this, value, element, true)
+        DigitsBetween: function(value, element, params) {
+            return ($.validator.methods.number.call(this, value, element, true)
                 && value.length>=parseFloat(params[0]) && value.length<=parseFloat(params[1]));
-        }, $.validator.format("The field must be beetwen {0} and {1} digits."));
+        },
 
         /**
          * Validate the size of an attribute.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelSize", function(value, element, params) {
-            return this.optional(element) ||
-                helpers.getSize(this, element,value) == params[0];
-        }, $.validator.format("The field must be {0}"));
+        Size: function(value, element, params) {
+            return laravelValidation.helpers.getSize(this, element,value) == params[0];
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/842620834/issues/source?bid=2225045&fileBranchId=2805859#l247)